charon/pkgs/npm.py

```diff
@@ -160,6 +165,31 @@ def handle_npm_uploading(
             failed_metas.extend(_failed_metas)
             logger.info("package.json uploading done")
 
+        if (key_id is not None or key_file is not None) and passphrase is not None:
+            artifacts = [s for s in valid_paths if s.endswith(".json") or s.endswith(".js")]
```

Comment:
Similar to maven. NPM package does not only mark "js" or "json" as valid artifacts.